src/System.Runtime/src/System.Runtime.Forwards.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.InteropServices.GCHandle))]
```

Comment:
What is the purpose of these type-forwards in the implementation? I assume it is to help with binary breaking changes between now and RTM. Is the goal to remove them be fore we ship?